notebooks/106-speech-recognition-quantize-wav2vec2/106-speech-recognition-quantize-wav2vec2.ipynb

```diff
@@ -0,0 +1,626 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "cacbe6b4",
+   "metadata": {
+    "id": "rQc-wXjqrEuR"
+   },
+   "source": [
+    "# Quantize speech recognition models with OpenVINO Post-Training Optimization Tool ​\n",
+    "This tutorial demonstrates how to apply INT8 quantization to the speech recognition model known as [Wav2Vec2](https://huggingface.co/docs/transformers/model_doc/wav2vec2), using the [Post-Training Optimization Tool API (POT API)](https://docs.openvinotoolkit.org/latest/pot_compression_api_README.html) (part of the [OpenVINO Toolkit](https://docs.openvinotoolkit.org/)). We will use a fine-tuned [Wav2Vec2-Base-960h](https://huggingface.co/facebook/wav2vec2-base-960h) [PyTorch](https://pytorch.org/) model trained on the [LibriSpeech ASR corpus](https://www.openslr.org/12). The tutorial is designed to be extendable to custom models and datasets. It consists of the following steps:\n",
+    "\n",
+    "- Download and prepare the Wav2Vec2 model and LibriSpeech dataset\n",
+    "- Define data loading and accuracy validation functionality\n",
+    "- Prepare the model for quantization\n",
+    "- Run optimization pipeline\n",
+    "- Compare performance of the original and quantized models"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4d6b41e6-132b-40da-b3b9-91bacba29e31",
+   "metadata": {},
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "771388d6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "import time\n",
+    "import re\n",
+    "import numpy as np\n",
+    "import torch\n",
+    "import tarfile\n",
+    "from pathlib import Path\n",
+    "from itertools import groupby\n",
+    "from addict import Dict\n",
+    "import soundfile as sf\n",
+    "\n",
+    "from transformers import Wav2Vec2ForCTC\n",
```

Comment:
If the intent is to use OpenVINO 2022.1 anyway, I would highly recommend using a new importing system for POT. You can find an example [here](https://github.com/openvinotoolkit/openvino/blob/master/tools/pot/openvino/tools/pot/api/samples/classification/classification_sample.py).